sdcm/nemesis.py

```diff
@@ -4935,3 +4964,18 @@ class StartStopValidationCompaction(Nemesis):
 
     def disrupt(self):
         self.disrupt_start_stop_validation_compaction()
+
+
+class FreeTierSetMonkey(SisyphusMonkey):
```

Comment:
@soyacz / @fruch, 
While working on nemesis.py I realized I don't understand why we need this MonkeySet.

Why don't we use SisyphusMonkey with nemesis_selector in the original test yaml?

Reply:
See
https://github.com/scylladb/scylla-cluster-tests/pull/5504#discussion_r1035603657

for the reason, it simply the configuration of multi-tenant setups.

Reply:
thx

Reply:
Looks like worth a comment in the code for that new nemesis.